python/ray/tune/examples/hyperopt_example.py

```diff
@@ -4,17 +4,19 @@
 
 import ray
 from ray.tune import run_experiments, register_trainable
-from ray.tune.hpo_scheduler import HyperOptScheduler
+from ray.tune.async_hyperband import AsyncHyperBandScheduler
```

Comment:
Can we move these into a schedulers/ folder?

Reply:
would prefer to do this in a separate PR